### ISSUE TITLE
Question about Training Material Diffusion Model from Scratch

### ISSUE THREAD

**Issue by angieAAAAA** [Peter-Kocsis/IntrinsicImageDiffusion#17]
Thanks for your wonderful work!

Is it possible to train the Material Diffusion model from scratch without loading any pre-trained weights? 

Any insights on this would be greatly appreciated!

**Comment by Peter-Kocsis** [comment]
Yes, please refer to our readme: https://github.com/Peter-Kocsis/IntrinsicImageDiffusion?tab=readme-ov-file#training
